toolkits/search/arcade_search/utils.py

```diff
@@ -51,6 +63,123 @@ def call_serpapi(context: ToolContext, params: dict) -> dict:
         )
 
 
+# ------------------------------------------------------------------------------------------------
+# Google Maps utils
+# ------------------------------------------------------------------------------------------------
+def get_google_maps_directions(
```

Comment:
Can we reduce the response payload size? This is returning more than just directions such as `search_parameters`, `search_metadata`, `places_info`, `durations`. Also do we need to return the long `geo_photo` URLs or even the `gps_coordinates` for each intermediary step, or others?

The tool response for a 15 minute drive was a 500 line JSON